src/mosquito/runners/run_at_most.cr

```diff
@@ -0,0 +1,16 @@
+module Mosquito::Runners
+  module RunAtMost
+    getter execution_timestamps = {} of Symbol => Time
+
+    private def run_at_most(*, every interval, label name, &block)
+      now = Time.utc
+      last_execution = @execution_timestamps[name]? || Time.unix 0
+      delta = now - last_execution
```

Comment:
Given we're measuring time here, might be better to use `Time.monotonic`? Ref: https://crystal-lang.org/api/Time.html#measuring-time

Reply:
Good call! Fun fact this code is older than Time.monotonic 🤪